biojava-core/src/test/java/org/biojava/nbio/core/sequence/io/embl/EmblReaderTest.java

```diff
@@ -0,0 +1,47 @@
+package org.biojava.nbio.core.sequence.io.embl;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * This class should test the parser of EmblReader class
+ * @since 5.0.0
+ * @author Noor Aldeen Al Mbaidin
+ */
+public class EmblReaderTest {
+
+
+    @Test(expected = NullPointerException.class)
+    public void givenNullFileParameterWhenProcessEmblFileThenThrowException() throws IOException {
+        File file = null;
+        EmblReader.process(file);
+
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void givenDirectoryWhenProcessEmblFileThenThrowException() throws IOException {
+        File file = new File("./src/test/resources");
```

Comment:
This is not quite right. See for instance how it is done here: https://github.com/biojava/biojava/blob/359dce819e7678879ae41301b11fa268f92e81d1/biojava-structure/src/test/java/org/biojava/nbio/structure/io/TestNonDepositedFiles.java#L60

The important thing is to use `getResourceAsStream` which will work in all situations (in jar packaging for instance).